howdoi/howdoi.py

```diff
@@ -332,30 +331,38 @@ def _get_links_with_cache(query):
 
 
 def _get_instructions(args):
+    """
+    @args: command-line arguments
+    returns: json string with answers and other metadata
+             False if unable to get answers
+    """
+
     question_links = _get_links_with_cache(args['query'])
     if not question_links:
         return False
 
-    only_hyperlinks = args.get('link')
-    star_headers = (args['num_answers'] > 1 or args['all'])
-
-    answers = []
-    initial_position = args['pos']
-    spliter_length = 80
-    answer_spliter = '\n' + '=' * spliter_length + '\n\n'
+    res = []
+    init_pos = args['pos']
+    num_answers = args['num_answers']
 
-    for answer_number in range(args['num_answers']):
-        current_position = answer_number + initial_position
-        args['pos'] = current_position
-        link = get_link_at_pos(question_links, current_position)
+    for inc_pos in range(num_answers):
+        curr_pos = inc_pos + init_pos
+        args['pos'] = curr_pos
+        link = get_link_at_pos(question_links, curr_pos)
```

Comment:
I feel like this is not related to the json issue (more of a code styling thing). Personally I like the more explicit variable names: `answer_number`, `current_position` and `initial_position`